### DISCUSSION TITLE
Support custom scrollbars in DataTable components

### DISCUSSION THREAD

**Discussion by iantrudell** [primefaces/community#495] [PrimeReact]
Similar to how the [ScrollPanel](https://primereact.org/scrollpanel/) component works, I would like to have the ability to use custom scrollbars in the DataTable component. Currently, only native browser scrollbars are supported via the scrollable option and thus have limited styling options.

**Comment by mertsincan** [comment]
Hi,

Sorry for the delayed response! Unfortunately, we do not plan to add this. DataTable will have a big impact on big data in terms of performance and rendering.

Best Regards,
